hathor/cli/run_node.py

```diff
@@ -201,7 +206,11 @@ def prepare(self, *, register_resources: bool = True) -> None:
 
         if self._args.stratum:
             assert self.manager.stratum_factory is not None
-            self.reactor.listenTCP(self._args.stratum, self.manager.stratum_factory)
+
+            if self._args.x_enable_ipv6:
+                self.reactor.listenTCP(self._args.stratum, self.manager.stratum_factory, interface='::0')
```

Comment:
Why `::0`? I understand this might be default but there should be a way to choose one specific interface.

Reply:
This is a good point. I'll add new parameters to allow changing this default.

Reply:
Done

Reply:
Thanks!